fdbserver/masterserver.actor.cpp

```diff
@@ -170,6 +170,10 @@ struct MasterData : NonCopyable, ReferenceCounted<MasterData> {
 		recoveryTransactionVersion;  // The first version in this epoch
 	double lastCommitTime;
 
+	Version liveCommittedVersion; // The live committed version reported by proxies.
```

Comment:
Better comment: The largest live committed version reported by proxies.

Reply:
Done.